### ISSUE TITLE
ARC_3_1_Additions.html not found

### ISSUE THREAD

**Issue by dihm** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#89]
It appears that ARC_3_1_Additions.html is not actually present on RTD (and the link to it on the release actually points to ARC_3_0_Introduction.html).

Would it be possible to add a short paragraph to [this page](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/getting_started.html) that links to the generated html and a download link for the ipynb?

**Comment by nikolasibalic** [comment]
Hi David @dihm  ,

I did quite an overhaul of [documentation](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/) now to make it more accessible (you might  need to refresh page cache to see newest version). In doing so I also fixed error you noticed in linking. I still have to add few clarification that 3.1 additions does not support  divalent atoms for now, but that should be it.

I hope this update makes it faster for everyone to find relevant bits.

Best wishes,
Nikola
